modules/script/script-impl/global.d.ts

```diff
@@ -0,0 +1,12 @@
+/*
+This project don't include XP libraries definitions and need these interfaces to properly
+map imports via require or __non_webpack_require__.
+
+Warning: all of these interfaces are empty. If you want type definitions for various libraries,
+you need to add types for there libraries, where those interfaces are redefined.
+*/
+interface XpEventLibrary {}
+
+interface XpLibraries {
+    '/lib/xp/event': XpEventLibrary;
+}
```

Comment:
I think it's cleaner to just merge `XpLibraries` in `global` right in the library files, instead of going via `XpEventLibrary` and similar.

Reply:
Yes, definitely. Guess, I overcomplicated it.